### PR TITLE
Add blankslates

### PR DESCRIPTION
**Pull request by briannelson**
Looks a little nicer. Also, added the same font stack Github is using for a more pleasing experience.

octobox

### CODE REVIEW

**Comment by briannelson** [comment]
I'm open to suggestions on the wording 😄

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,9 +1,11 @@
 $navbar-default-bg: #312312;
 $table-bg-active: #fafafa;
 $tooltip-opacity: 1;
+$font-family-sans-serif: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```
Comment (briannelson) [inline app/assets/stylesheets/application.scss]: Yup, same font stack as Github.com

**Comment by andrew** [comment]
Sweet!

**Comment by andrew** [comment]
Got some conflicts here but looks great!

**Comment by briannelson** [comment]
Should be all set @andrew

**Comment by andrew** [comment]
Wonderful, thanks again @briannelson!

**Pull request merged**